Autopilot/Path Manager/MPL3115A2.h

```diff
@@ -29,15 +29,16 @@ extern "C" {
 #define BAR_IN_LSB 0x15
 #define CONTROL_REGISTER1 0x26
 #define OFF_H 0x2D //offset register for alttitude
-#define GIVEN_PRESSURE 97913 //current pressure
+#define GIVEN_PRESSURE 97561 //current pressure
 
 
 
 //TODO: ADD BETTER FORMATTING TO THIS HEADER FILE
 char initAltimeter();
-void calibrateAltimeter(long int pressure);
+void calibrateAltimeter(int altitudeOffset);
 float getAltitude();
-
+void switchToBarometerMode(int toBarometer);
```

Comment:
this function will never be used externally (only internally within the driver implementation). Therefore it should not exist in the .h file, and should only exist within the scope of the c file as a static function to keep it private.